src/wdic_game/scene_manager.py

```diff
@@ -27,9 +27,14 @@ def run(self):
                     self.running = False
                 else:
                     self.scene.handle_event(event)
+                    font = pygame.font.Font(None, 20)
```

Comment:
Der Punktestand wird jetzt in allen Szenen angezeigt. Momentan kommt man aus der Game Scene nicht mehr heraus, dann spielt es keine Rolle. "Schöner" wäre den Punktestand direkt in der `render` Methode der Gamescene anzuzeigen:
https://github.com/semiversus/WDICGame/blob/ab64b2a54f05b8f6f3f9036ceef1e39c2adb1fcb/src/wdic_game/game.py#L40